tests/cases/fourslash/jsDocContextualTagsInJsDoc1.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path="fourslash.ts" />
+
+//// /**
+////  * @param x/*1*/ Does the thing
+////  */
+//// function foo(x/*2*/) {}
+
+verify.quickInfoAt("1", "(parameter) x: any", "Does the thing", undefined);
+verify.quickInfoAt("2", "(parameter) x: any", "Does the thing", [{ name: "param", text: "x Does the thing" }]);
```

Comment:
I think that this isn't particularly desirable result either. It still contains duplicate information. I think this should be fixed as part of https://github.com/microsoft/TypeScript/issues/57819 though and not here